doc/zmq.md

```diff
@@ -102,7 +104,17 @@ is structured as the following based on the type of message:
 
 Where the 8-byte uints correspond to the mempool sequence number.
 
-These options can also be provided in bitcoin.conf.
+`rawtx`: notifies when a transaction is added to mempool. The body of the message is the raw transaction in hexadecimal format.
+
+`hashtx`: notifies when a transaction is added to mempool. The body of the message is structured in the following way:
+
+    hashtx <32-byte transaction hash>
+
+`rawblock`: notifies when a the block tip is updated. The body of the message is the raw block in hexadecimal format.
+
+`hashblock`: notifies when a the block tip is updated. The body of the message is structured in the following way:
+
+    hashblock <32-byte block hash>
```

Comment:
see comments for `hashtx` above (remove hashblock and mention Little Endianess)

Reply:
Done.